src/lib/sql/sql_translator.cpp

```diff
@@ -1520,10 +1530,26 @@ std::shared_ptr<AbstractExpression> SQLTranslator::_inverse_predicate(const Abst
   Fail("GCC thinks this is reachable");
 }
 
+std::vector<std::shared_ptr<AbstractExpression>> SQLTranslator::_retrieve_expressions(
```

Comment:
The name `_retrieve_expressions` doesn't tell me too much. Maybe `_unwrap_named_expressions`?